client/public/elements/pages/components/app-components.tpl.js

```diff
@@ -11,6 +13,7 @@ import "../../components/download-list"
 import "../../components/dropdown"
 import "../../components/hero-image"
 import "../../components/icon"
+import "../../components/infographic"
```

Comment:
For the other RP, but the infographic page name was to include more then the factoid, so this should probably be called rp-factoid